defaults/openstack/config.pan

```diff
@@ -4,48 +4,126 @@ unique template defaults/openstack/config;
 # Define site specific variables #
 ##################################
 include if_exists('site/openstack/config');
+@use{
+  type = ipaddress
+  note = This is the primary IP address of the Host
+}
 variable PRIMARY_IP ?= DB_IP[escape(FULL_HOSTNAME)];
```

Comment:
Is there any reason to not rename it OPENSTACK_PRIMARY_IP ? to match variable name convention ?